components/layout/model.rs

```diff
@@ -507,61 +522,59 @@ impl ToGfxMatrix for ComputedMatrix {
 // https://drafts.csswg.org/css2/visudet.html#min-max-widths
 // https://drafts.csswg.org/css2/visudet.html#min-max-heights
 /// A min or max constraint
-///
-/// A `max` of `None` is equivalent to no limmit for the size in the given
-/// dimension. The `min` is >= 0, as negative values are illegal and by
-/// default `min` is 0.
 #[derive(Debug)]
-pub struct MinMaxConstraint {
-    min: Au,
-    max: Option<Au>
+pub struct SizeConstraint {
```

Comment:
(It'd be nice to keep the renaming in a different commit, but that doesn't matter now :P)